.github/workflows/pipeline-tasks.yaml

```diff
@@ -0,0 +1,42 @@
+name: pipeline-tasks
```

Comment:
what task is it?

Reply:
Fourth task. Creating pipelines.

Reply:
this file already done by other student and will not be estimated. Please close PR and start next task.

Reply:
This is completely my file there is no way someone could make it. I also added this file and tested it running on my other repository about ten hours ago. Here is the proofs: https://github.com/NazarKarpiuk/test_repos/tree/main/.github/workflows